PWGCF/FEMTOSCOPY/macros/AddTaskNanoPt.C

```diff
@@ -0,0 +1,271 @@
+//PT for beginning of ProtonDeuteron
+#include <vector>
+#include "AliAnalysisTaskSE.h"
+#include "AliAnalysisManager.h"
+#include "AliFemtoDreamEventCuts.h"
+#include "AliFemtoDreamTrackCuts.h"
+#include "AliFemtoDreamCollConfig.h"
+#include "AliAnalysisTaskNanoPt.h"
+
+AliAnalysisTaskSE* AddTaskNanoPt(  bool isMC = false,
+                                   TString trigger = "kINT7",
+                                   bool DCAPlots = false,//3
+                                   bool CombSigma = false,//4
+                                   bool ContributionSplitting = false)
+{
+ 
+
+ 
+  AliAnalysisManager *mgr = AliAnalysisManager::GetAnalysisManager();
+
+  if (!mgr)
+  {
+    printf("No analysis manager to connect to!\n");
+    return nullptr;
+  }
+  
+  if (!mgr->GetInputEventHandler()) 
+  {
+    printf("This task requires an input event handler!\n");
+    return nullptr;
+  }
+
+
+//========= Init subtasks and start analyis ============================
+// Event Cuts
+  AliFemtoDreamEventCuts *evtCuts = AliFemtoDreamEventCuts::StandardCutsRun2();
+  evtCuts->CleanUpMult(false, false, false, true);
+
+// =====================================================================
+  
+
+  //Proton track Cuts----------------------------------------------------------------------------
+ AliFemtoDreamTrackCuts *TrackCuts =
+    AliFemtoDreamTrackCuts::PrimProtonCuts(isMC, true, CombSigma, ContributionSplitting);
+ 
+     TrackCuts->SetMinimalBooking(false);
```

Comment:
also formatting (I know this is picky but in the long run it will massively help to maintain the readability of your code).
The formatter (in your editor) will massively help here